cit3.0-web/src/components/AddMarker/ResourceMarker.js

```diff
@@ -27,5 +27,6 @@ export default function ResourceMarker({ resourceName, resources }) {
 }
 ResourceMarker.propTypes = {
   resourceName: PropTypes.string.isRequired,
-  resources: PropTypes.arrayOf(PropTypes.shape).isRequired,
+  resources: PropTypes.oneOfType([PropTypes.string, PropTypes.shape()])
```

Comment:
does this still count as an ArrayOf or was that part not needed?

Reply:
That part was old and outdated